specification/monitor/resource-manager/readme.md

```diff
@@ -60,6 +60,7 @@ input-file:
 - Microsoft.Insights/stable/2019-03-01/metricBaselines_API.json
 - Microsoft.Insights/stable/2018-03-01/metricAlert_API.json
 - Microsoft.Insights/preview/2022-08-01-preview/scheduledQueryRule_API.json
+- Microsoft.Insights/preview/2023-03-15-preview/scheduledQueryRule_API.json
```

Comment:
we should never keep two versions of the same swagger in one tag. Please remove the old one and same for other changes in this file.